vie.ini

```
@@ -3,10 +3,442 @@
 ; A message doesn't need to be there to be shown correctly on the client.
 ; Only messages shown in PocketMine itself need to be here
 
+language.name=Tiếng Việt
+language.selected=Đã chọn {%0} ({%1}) là ngôn ngữ cơ bản
 
+ability.flight=Bay
+ability.noclip=No-clip
+chat.type.achievement={%0} has just earned the achievement {%1}
+chat.type.admin=[{%0}: {%1}]
+chat.type.announcement=[{%0}] {%1}
+chat.type.emote=* {%0} {%1}
+chat.type.text=<{%0}> {%1}
+commands.ban.success=Đã cấm người chơi {%0}
+commands.ban.usage=/ban <tên người chơi> [lý do ...]
+commands.banip.invalid=Bạn đã nhập một địa chỉ IP không hợp lệ hoặc người chơi đó không trực tuyến
+commands.banip.success.players=Banned IP address {%0} belonging to {%1}
+commands.banip.success=Đã cấm địa chỉ IP {%0}
+commands.banip.usage=/ban-ip <địa chỉ IP|tên người chơi> [lý do ...]
+commands.banlist.ips=Có tất cả {%0} địa chỉ IP đã bị cấm:
+commands.banlist.players=There are {%0} total banned players:
+commands.banlist.usage=/banlist [ips|players]
+commands.clear.failure.no.items=Could not clear inventory of {%0}, no items to remove
```

```suggestion
commands.clear.failure.no.items=Không thể xóa túi đồ của người chơi {%0} vì không có bất kỳ vật phẩm nào để xóa bỏ
```
`clear` : `dọn`, `xóa` or `dọn dẹp` ?
`inventory` : `túi đồ`, `kho đồ` or `rương đồ` ?
`remove` : `xóa`, `loại bỏ` or `gỡ bỏ` ?